content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 1. While
+taxonomie:
+  - ib-19.2.Controlestructuur-Herhaling-While.OI
+  - ib-19.3.Controlestructuur-Herhaling-While.OI
+draft: false
+---
+
+## Wat is een While-loop?
+In de programmeerwereld worden **While-loops** gebruikt om een bepaald stukje code herhaaldelijk uit te voeren zolang aan een voorwaarde wordt voldaan. Dit maakt het mogelijk om dynamische herhalingen te creëren, waarbij je vooraf niet weet hoe vaak de loop zal draaien. De herhaling stopt zodra de voorwaarde **onwaar** (false) wordt.
```

Comment:
Suggestie: 'Dit' veranderen naar iets in zin van 'Samengenomen maakt dit'. 
'Dit' klinkt prima, al is het voor een nieuwkomer mogelijk onduidelijk naar welk deel in voorafgaande zin het verwijst. 

Suggestie: tussen haakjes (altijd) toevoegen tussen niet en weet.

In andere items/documenten/onderdelen(?) werd overigens vooral over condities gesproken i.p.v. voorwaarden, iirc. Denk niet dat dat veel uitmaakt, tho.